hasher-matcher-actioner/hmalib/common/message_models.py

```diff
@@ -208,9 +220,12 @@ def could_be(cls, d: dict) -> bool:
         """
         return "EventType" in d
 
+    def get_bytes(self) -> bytes:
+        return URLImageSource().get_image_bytes(self.url)
+
 
 @dataclass
-class S3ImageSubmission:
+class S3ImageSubmission(ImageSubmission):
```

Comment:
Could probably combine the `Submission` and `Source` objects such as `S3ImageSubmission` and `S3BucketImageSource`